src/index.ts

```diff
@@ -5,11 +5,7 @@
  * -------------------------------------------------------------------------------------------
  */
 
-import { init } from "./PolyFill";
-/**
- * Polyfilling Promise and fetch if not present
- */
-init();
+import "./PolyFill";
```

Comment:
I recommend not including the polyfils for modules as they are usualy unnecessary. This is causing a weird error for me when running in Chrome and removing this line fixes it.

Reply:
Not including polyfill will be a breaking change, since this makes user to include his own polyfill in case if its not natively present. We will eventually remove this in some time.